extensions/wikia/Places/js/Places.js

```diff
@@ -67,7 +67,8 @@ var Places = Places || (function(){
 			lon: lon,
 			width: $('#WikiaArticle img').eq(0).width(),
 			align: $('#WikiaArticle figure').eq(0).css('float'),
-			articleId: window.wgArticleId
+			articleId: mw.config.get('wgArticleId'),
+			token: mw.user.tokens.get('editToken')
 		};
 
 		// TODO: add progress indicator
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)